src/main/java/Duke.java

```diff
@@ -1,10 +1,23 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner sc = new Scanner(System.in);
+        TaskManager taskManager = new TaskManager();
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        String nextLine = sc.nextLine();
+        LineParser lineParser = new LineParser(nextLine);
+        Command command = lineParser.getTypeOfCommand();
+
+        while (!taskManager.getTaskManagerSaysBye()) {
```

Comment:
Perhaps the name function getTaskManagerSaysBye() can be changed to something like hasTaskManagerSaidBye to make it clearer as to what the function is doing.